.openshift-applier/templates/omp-be/deploy.yml

```diff
@@ -0,0 +1,128 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: http-app-deploy
+  annotations:
+    openshift.io/display-name: omp-be deploy
+    description: A template to deploy open management portal backend
+    iconClass: icon-cube
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: "${NAME}"
+    name: "${NAME}"
+  spec: {}
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    name: "${NAME}"
+    labels:
+      app: "${NAME}"
+  spec:
+    replicas: 1
+    selector:
+      name: "${NAME}"
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          name: "${NAME}"
+      spec:
+        containers:
+        - envFrom:
+            - secretRef:
+                name: mongo
+        - image: "${NAME}"
+          imagePullPolicy: Always
```

Comment:
`envFrom` should be included under `image`: here. Having it seperate like you do above causes it to not be mounted into the container (which then causes the db connection to fail and the container enters a crashloop)

Reply:
Nice catch! I thought I had updated that, but must not have committed it. I'll update that now.

Reply:
Fixed!